iFixit/react-commerce
# chore(deps): bump strapi to `v4.12.1`

Closes https://github.com/iFixit/react-commerce/issues/1867

## QA

Make sure strapi still works and is on the correct version, and smoke test product pages and product list pages.

Try running `pnpm install:all` locally and confirm you don't get a diff.
